supernovacontroller/sequential/gpio.py

```diff
@@ -0,0 +1,231 @@
+from transfer_controller import TransferController
+from BinhoSupernova.Supernova import Supernova
+from BinhoSupernova.commands.definitions import (
+    GetUsbStringSubCommand, GpioPinNumber, GpioLogicLevel, GpioFunctionality, GpioTriggerType,
+    COMMANDS_DICTIONARY, GPIO_CONFIGURE_PIN, GPIO_DIGITAL_WRITE, GPIO_DIGITAL_READ,
```

Comment:
These constants seem not to be used in the code, can you confirm this? If so, remove them from the list of imported items.